packages/nx/src/lock-file/npm.spec.ts

```diff
@@ -38,6 +39,13 @@ const YargsAndDevkitPackage = {
   version: '0.0.0',
   dependencies: { '@nrwl/devkit': '15.0.13', yargs: '17.6.2' },
 };
+const Ssh2Package = {
+  name: 'test',
+  version: '0.0.0',
+  dependencies: {
+    ssh2: '1.11.0',
+  },
+};
 
 describe('npm LockFile utility', () => {
   describe('v3', () => {
```

Comment:
Can you please also add a test for V1, or at least replace the V3 with backward-compatible V2?